src/sales/billing-agreements.md

```diff
@@ -15,15 +15,64 @@ _Billing agreements_
 
 1. Find the billing agreement in the list.
 
-1. In the _Actions_ column, click **View**.
+1. Click the needed billing agreement to be viewed.
```

Comment:
Combine these steps to one like the other following procedures:

`1. Find the billing agreement in the list and click to open it.`